test/run_pass/test_pwm.js

```diff
@@ -19,12 +19,17 @@ var Pwm = require('pwm');
 
 var pwm = new Pwm();
 
-var configuration = {};
+var configuration = {
+  period: 0.001,  // 1kHz
+  dutyCycle: 0.2  // 20%
+};
 
 if (process.platform === 'linux') {
   configuration.pin = 0;
 } else if (process.platform === 'nuttx') {
   configuration.pin = require('stm32f4dis').pin.PWM1.CH1_1;
+} else if (process.platform === 'tizenrt') {
+  configuration.pin = require('artik053').pin.PWM0;
```

Comment:
Memory usage will increase. In addition, developers should be include additional module. I think, this will decrease usability. The value in the pin module contains information for initializing pwm on stm. It includes pin number, port number, pin mode, and so on, not 1 2 3. The pin module of stm are required for advanced developers. However, I plan to support pwm for general developers without including pin modules.